hub/powertoys/file-explorer.md

```diff
@@ -23,27 +23,38 @@ File Explorer add-ons currently include:
   - G-code files (.gcode)
   - STL files (.stl)
 
-## Preview Pane
 
-Preview Pane is an existing feature in the Windows File Explorer which shows a lightweight, rich, read-only preview of the file's contents in the view's reading pane. PowerToys adds four extensions: Markdown, SVG, PDF, and G-code. PowerToys also adds support for source code files (for more than 150 file extensions).
+## Thumbnail preview
+
+Showing thumbnails is a built-in Windows feature. For thumbnail preview, PowerToys adds multiple extensions: SVG, PDF, G-code and STL.
+
+### Enabling PowerToys Thumbnail previews
+
+To enable, set which you would like enabled to **On**.
+
+## Preview Pane previewers in PowerToys
 
-### Enabling Preview Pane
+Preview Pane is an existing feature in the Windows File Explorer which allows you to see a preview of the file's contents in the view's reading pane. PowerToys adds multiple extensions: Markdown, SVG, PDF, and G-code. In addition to those, PowerToys also adds support for source code files (for more than 150 file extensions).
 
-To enable, first set all to **On**.
+### Enabling PowerToys Preview Pane previewers
+
+To enable, set which you would like enabled to **On**.
```

Comment:
```suggestion
To enable, set those which you would like enabled to **On**.
```

Reply:
@mattwojo 'set those which' sounds awkward to me, thoughts?

Reply:
set which items you ....?

Reply:
Updating to: "To enable preview support, set the extension to **On**."